src/cmd-init

```diff
@@ -124,17 +123,17 @@ mkdir -p src
                 git -C ./config fetch origin "$COMMIT"
                 git -C ./config reset --hard "$COMMIT"
             fi
-            if [ -n "${subdir}" ]; then
-                mv config config-git
-                ln -sr config-git/"${subdir}" config
-            fi
             (set +x; cd config && echo -n "Config commit: " && git describe --tags --always --abbrev=42)
             ;;
      esac
-     manifest=config/manifest.yaml
-     if ! [ -f "${manifest}" ]; then
-         echo 1>&2 "Failed to find src/${manifest}"
-         fatal "If using a custom configuration, be sure it has a manifest.yaml."
+     manifest="config/manifest.yaml"
+     if [[ ! -f "${manifest}" ]]; then
+         versionscript="config/select_version.sh"
```

Comment:
We're requiring a script to be present but not executing it?  I don't quite understand that.

In general I think we should try to be declarative where possible, and use code execution where needed.  (That's what a whole lot of things are, from make to ansible to kubernetes pod specs, etc.)

Reply:
👍🏻 . I've reworked the logic.